jakelib/pipeline.jake

```diff
@@ -52,22 +52,21 @@ namespace('pipeline', function () {
     const username = getUsername();
 
 		const cmds = [
-      `aws lambda invoke --function-name ${YMIR_ARN} --invocation-type Event --payload '{"requester":"${username}","repo_name":"${repo}","origin_branch":"${toBranch}","destination_branch":"${cluster}", "optional_env":"${environment}"}' response.json`
+      `aws lambda invoke --function-name ${YMIR_ARN} --cli-binary-format raw-in-base64-out --invocation-type Event --payload '{"requester":"${username}","repo_name":"${repo}","origin_branch":"${toBranch}","destination_branch":"${cluster}", "optional_env":"${environment}"}' response.json`
     ];
     console.log('If the execution was successful, you will get a 202 response code below:');
     jake.exec(cmds, { printStdout: true });
   });
   
   desc('Refresh via pipeline. | [\'cluster\',optional:\'ephemeral stack\']');
 	task('refresh', ['aws:loadCredentials'], { async: true }, function(cluster, ephStack) {
-
     const allowed_clusters = Object.keys(CLUSTERS.stag)
     allowed_clusters.push('eph');
```

Comment:
the indentation looks a bit odd here.  maybe both spaces and tabs?

Reply:
you're probably right -- I'll clean up in a separate PR